.github/workflows/label-checker.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: Label Checker
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - labeled
+      - unlabeled
+
+
+jobs:
+
+  check_labels:
+    name: Check labels
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Labels and Add Comment
+        id: check_labels_and_comment
+        uses: actions/github-script@v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            // Get PR info: id and labels
+            const prNumber = context.payload.pull_request.number;
+            const labels = context.payload.pull_request.labels.map(label => label.name);
+
+            // Array of labels to check for
+            const validLabels = ['enhancement', 'deprecated', 'refactoring', 'pr: breaking', 'pr: clean', 'pr: fix', 'pr: new feature', 'pr: test'];
```

Comment:
This is a good idea but are you sure about the list of labels required?

I would have expected a first list with the status:
to review || wip 
And a second list on the type:
pr: xxx

btw I would turn `refactoring` into `pr: refactoring`   and same for `pr: enhancement`
And I would remove deprecated from this list